docs/README-AWS.md

```diff
@@ -1,829 +1,20 @@
 # Introduction
 
-This readme file provides initial documentation for the Elastic Serverless Forwarder, an Amazon Web Services (AWS) Lambda function that ships logs from an AWS environment to Elastic.
+The Elastic Serverless Forwarder is an Amazon Web Services (AWS) Lambda function that ships logs from an AWS environment to Elastic.
 
-## Table of contents
-
-- [Overview](#overview)
-- [Inputs](#inputs)
-- [Deploying Elastic Serverless Forwarder](#deploying-elastic-serverless-forwarder)
-- [Permissions and policies](#permissions-and-policies)
-- [Sample S3 config file](#sample-s3-config-file)
-- [Using Secrets Manager](#using-secrets-manager)
-- [Using tags](#using-tags)
-- [Defining include/exclude filters](#defining-includeexclude-filters)
-- [JSON content discovery](#json-content-discovery)
-- [Expanding events from list in json object](#expanding-events-from-list-in-json-object)
-- [Manage multiline messages](#manage-multiline-messages)
-- [Routing AWS Service Logs](#routing-aws-service-logs)
-- [Setting up S3 event notifications for SQS](#setting-up-s3-event-notifications-for-sqs)
-- [Troubleshooting and error handling](#troubleshooting-and-error-handling)
-- [Resources and links](#resources-and-links)
+Please refer to the official [Elastic documentation for Elastic Serverless Forwarder](https://www.elastic.co/guide/en/observability/master/aws-elastic-serverless-forwarder.html) for detailed instructions on how to deploy and configure the forwarder.
```

Comment:
> https://www.elastic.co/guide/en/observability/master/aws-elastic-serverless-forwarder.html

When we'll release 1.6.0 of the forwarder can we pin to something like https://www.elastic.co/guide/en/observability/1.6.0/aws-elastic-serverless-forwarder.html?

If I got it right https://www.elastic.co/guide/en/observability/%VERSION% is the stack version, is it?

I find it troublesome in such case.
We'll push updated documentation for ESF 1.6.0 and probably backport to every stack versions the forwarder is compatibile with.
Where will the docs for ESF 1.5.0 end up? lost?

is https://www.elastic.co/guide/en/observability/master/aws-elastic-serverless-forwarder.html the docs for the "main" (master) branch of ESF or ESF compatible with master branch of the stack?
In the second case there will be multiple ESF versions compatibile, which ESF version will be the one effectively published at that URL?

How do I keep consistent the link across README sent to SAR for a specific ESF version to the documentation of that specific ESF version on https://www.elastic.co/guide?

I can see it can be done only if for https://www.elastic.co/guide/en/observability/%VERSION%, %VERSION% refers to the ESF version

Reply:
> If I got it right https://www.elastic.co/guide/en/observability/%VERSION% is the stack version, is it?

Yes, that is stack version.
Hmm, interesting problem (and not unique, though I don't know how well our systems are currently set for such a use case...). Some questions to work out the way forward:

1. Do you see users staying on older versions of ESF for a period of time or will they generally upgrade as soon as new version is available? How will they know a new version is released?
2. Do we actually need to keep the ESF docs versioned independently? i.e. keep version 1.5 readily available for compairson and for users even when 1.6 is released? Obviously previous versions will be available via GitHub source but not as nicely/usable as choosing a version from main docs
3. Would a link to the GitHub branch for that version of the ESF docs suffice? Or a tagged version via GitHub?

Generally I've solved this problem in the past by having packages with independent versioning being pulled into a super-docset that has its own versioning. From within the equivalent of main or 8.2 etc. you can see the latest version of the package/forwarder and then independently choose between 'sub' versions i.e. 1.5 of the ESF docs.

I can discuss with my team but we might have to consider NOT cutting back the docs in this case and keeping the README docs as independently versioned within this repo, and only publishing the latest version via /guide. It's not ideal and we might still have to use branches or tags within this repo to indicate the correct version to SAR.

Let me know what you think to questions and I will find out a bit more about what is possible or how we could build towards a more elegant solution...